keyboards/plywrks/lune/info.json

```
@@ -0,0 +1,75 @@
+{
+    "keyboard_name": "Lune",
+    "url": "",
+    "maintainer": "Ramon Imbao",
```

This field is meant to reference your GitHub username.

```suggestion
    "maintainer": "ramonimbao",
```